app/assets/stylesheets/getting-started.scss

```diff
@@ -1,7 +1,11 @@
 #hello-there {
-  .avatar {
-    width: 200px;
-    height: 200px;
+  .well {
+    padding-bottom: 30px;
+  }
```

Comment:
I neither said nor thought that. Your PR would be a correct solution for the problem. I'd just prefer changing the mentioned file to make Bootstrap apply the correct styles for us.